src/jit/compiler.h

```diff
@@ -6844,10 +6850,15 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     void unwindReserveFunc(FuncInfoDsc* func);
     void unwindEmitFunc(FuncInfoDsc* func, void* pHotCode, void* pColdCode);
 
-#if defined(_TARGET_AMD64_)
+#ifdef _TARGET_XARCH_
```

Comment:
@janvorli As you pointed out, the implementation of these functions exists only when ``_TARGET_AMD64 || (_TARGET_X86 && FEATURE_EH_FUNCLET)``.  77dc419 revised #ifdef condition.